util/chunk/row_container.go

```diff
@@ -229,6 +251,12 @@ func (a *SpillDiskAction) Action(t *memory.Tracker) {
 		logutil.BgLogger().Info("memory exceeds quota, spill to disk now.",
 			zap.Int64("consumed", t.BytesConsumed()), zap.Int64("quota", t.GetBytesLimit()))
 	})
+	if atomic.LoadUint32(&a.c.readOnly) == 1 {
+		err := a.c.spillToDisk()
```

Comment:
`memory exceeds quota, spill to disk now.`

This log is missing

Reply:
It logs when the rowContainer is exceeded.